src/api/dependency-graph/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "dependency-graph",
+  "private": true,
+  "version": "0.1.0",
+  "description": "A dependency graph compilation service for Telescope",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node src/server.js"
```

Comment:
Let's also add `nodemon` like the other services do for dev, and include a way to pass a local set of environment variables.  Steal code from another service that does this.